### ISSUE TITLE
Error running update session

### ISSUE THREAD

**Issue by AnzhiZhang** [Lazy-Bing-Server/RegionFileUpdaterMulti-MCDR#4]
![image](https://github.com/user-attachments/assets/dffde9c3-746a-4cdc-ac91-c985de9182be)
![image](https://github.com/user-attachments/assets/ede21f81-6e9d-454e-afe4-8711b4e99359)

**Comment by Ra1ny-Yuki** [comment]
Can't reproduce in my environment, provide more details pls

**Comment by AnzhiZhang** [comment]
MCDR 3.13.2
Python 3.12.7
https://pastebin.com/psfirpWL

**Comment by Ra1ny-Yuki** [comment]
Maybe this works: [45f94d8](https://github.com/Lazy-Bing-Server/RegionFileUpdaterMulti-MCDR/tree/45f94d8738b9c3f984a42ef40e60830a1fc36397)

**Comment by AnzhiZhang** [comment]
```text
[MCDR] [22:06:39] [MainThread/INFO]: 服务端已关闭
[PB] [2024-11-16 22:06:39,724 INFO] Storage root set to '/survival_lfs/pb_files'
[PB] [2024-11-16 22:06:39,732 INFO] Found latest non-temp backup #647
[PB] [2024-11-16 22:06:39,734 INFO] Found file FileInfo(backup_id=647, path='world/region/r.1.1.mca', mode=33279, content=None, blob=BlobInfo(hash='4724fc2e2644b9971c15fccf2f552b39', compress=CompressMethod('lz4'), raw_size=6385664, stored_size=6344404, file_count=0), uid=0, gid=0, ctime_ns=1731142688189217364, mtime_ns=1731142688189217364, atime_ns=1731599324896079520)
[PB] [2024-11-16 22:06:39,734 INFO] Exporting child 'world/region/r.1.1.mca' in Backup(id=647, timestamp=1731675654513174815, creator='player:Andy_Zhang', comment='', targets=['world'], tags={}, file_raw_size_sum=16008023632, file_stored_size_sum=12497276800) to directory server/world/region, recursively = False
[PB] [2024-11-16 22:06:40,137 INFO] Export done
[MCDR] [22:06:40] [RFUMulti@ComplexPayloadExecutor_0/INFO] [region_file_updater_multi]: - [PrimeBackupUpstream] <survival_pb> region/r.1.1.mca -> ./server/world
[PB] [2024-11-16 22:06:40,467 INFO] Storage root set to '/survival_lfs/pb_files'
[PB] [2024-11-16 22:06:40,475 INFO] Found latest non-temp backup #647
[PB] [2024-11-16 22:06:40,477 INFO] Found file FileInfo(backup_id=647, path='world/poi/r.1.1.mca', mode=33279, content=None, blob=BlobInfo(hash='8fe9dcc7387777d346fa75e1cdb86444', compress=CompressMethod('lz4'), raw_size=602112, stored_size=6956, file_count=0), uid=0, gid=0, ctime_ns=1730423783261200823, mtime_ns=1730421541490030807, atime_ns=1731142489754682943)
[PB] [2024-11-16 22:06:40,477 INFO] Exporting child 'world/poi/r.1.1.mca' in Backup(id=647, timestamp=1731675654513174815, creator='player:Andy_Zhang', comment='', targets=['world'], tags={}, file_raw_size_sum=16008023632, file_stored_size_sum=12497276800) to directory server/world/poi, recursively = False
[PB] [2024-11-16 22:06:40,867 INFO] Export done
[MCDR] [22:06:40] [RFUMulti@ComplexPayloadExecutor_0/INFO] [region_file_updater_multi]: - [PrimeBackupUpstream] <survival_pb> poi/r.1.1.mca -> ./server/world
[PB] [2024-11-16 22:06:41,197 INFO] Storage root set to '/survival_lfs/pb_files'
[PB] [2024-11-16 22:06:41,205 INFO] Found latest non-temp backup #647
[PB] [2024-11-16 22:06:41,207 ERROR] File 'world/entities/r.1.1.mca' in backup #647 does not exist
[MCDR] [22:06:41] [RFUMulti@ComplexPayloadExecutor_0/INFO]: 正在启动服务端
```

**Comment by Ra1ny-Yuki** [comment]
> ```
> [MCDR] [22:06:39] [MainThread/INFO]: 服务端已关闭
> [PB] [2024-11-16 22:06:39,724 INFO] Storage root set to '/survival_lfs/pb_files'
> [PB] [2024-11-16 22:06:39,732 INFO] Found latest non-temp backup #647
> [PB] [2024-11-16 22:06:39,734 INFO] Found file FileInfo(backup_id=647, path='world/region/r.1.1.mca', mode=33279, content=None, blob=BlobInfo(hash='4724fc2e2644b9971c15fccf2f552b39', compress=CompressMethod('lz4'), raw_size=6385664, stored_size=6344404, file_count=0), uid=0, gid=0, ctime_ns=1731142688189217364, mtime_ns=1731142688189217364, atime_ns=1731599324896079520)
> [PB] [2024-11-16 22:06:39,734 INFO] Exporting child 'world/region/r.1.1.mca' in Backup(id=647, timestamp=1731675654513174815, creator='player:Andy_Zhang', comment='', targets=['world'], tags={}, file_raw_size_sum=16008023632, file_stored_size_sum=12497276800) to directory server/world/region, recursively = False
> [PB] [2024-11-16 22:06:40,137 INFO] Export done
> [MCDR] [22:06:40] [RFUMulti@ComplexPayloadExecutor_0/INFO] [region_file_updater_multi]: - [PrimeBackupUpstream] <survival_pb> region/r.1.1.mca -> ./server/world
> [PB] [2024-11-16 22:06:40,467 INFO] Storage root set to '/survival_lfs/pb_files'
> [PB] [2024-11-16 22:06:40,475 INFO] Found latest non-temp backup #647
> [PB] [2024-11-16 22:06:40,477 INFO] Found file FileInfo(backup_id=647, path='world/poi/r.1.1.mca', mode=33279, content=None, blob=BlobInfo(hash='8fe9dcc7387777d346fa75e1cdb86444', compress=CompressMethod('lz4'), raw_size=602112, stored_size=6956, file_count=0), uid=0, gid=0, ctime_ns=1730423783261200823, mtime_ns=1730421541490030807, atime_ns=1731142489754682943)
> [PB] [2024-11-16 22:06:40,477 INFO] Exporting child 'world/poi/r.1.1.mca' in Backup(id=647, timestamp=1731675654513174815, creator='player:Andy_Zhang', comment='', targets=['world'], tags={}, file_raw_size_sum=16008023632, file_stored_size_sum=12497276800) to directory server/world/poi, recursively = False
> [PB] [2024-11-16 22:06:40,867 INFO] Export done
> [MCDR] [22:06:40] [RFUMulti@ComplexPayloadExecutor_0/INFO] [region_file_updater_multi]: - [PrimeBackupUpstream] <survival_pb> poi/r.1.1.mca ->./server/world
> [PB] [2024-11-16 22:06:41,197 INFO] Storage root set to '/survival_lfs/pb_files'
> [PB] [2024-11-16 22:06:41,205 INFO] Found latest non-temp backup #647
> [PB] [2024-11-16 22:06:41,207 ERROR] File 'world/entities/r.1.1.mca' in backup #647 does not exist
> [MCDR] [22:06:41] [RFUMulti@ComplexPayloadExecutor_0/INFO]: 正在启动服务端
> ```

If you are sure the specified entities MCA file does exists, try to extract this file from this backup with PB CLI manually (with command `python PrimeBackup-v1.8.3.pyz extract [-h] [-o OUTPUT] [-r] backup_id file`) to ensure your PB works fine. . At least, the path `world/entities/r{x}.{z}.mca` provided here seems correct.

If manual extraction result is the same, this plugin works correctly as intended.

**Comment by AnzhiZhang** [comment]
The file does not exist. It works after the removal of `entities` from the config. Looking forward to a release.

**Comment by Ra1ny-Yuki** [comment]
Released:  [v1.0.3](https://github.com/Lazy-Bing-Server/RegionFileUpdaterMulti-MCDR/releases/tag/1.0.3)

This issue will be closed. If you found any problem still exists in new version, just reopen this at your will.
